.travis.yml

```diff
@@ -12,9 +15,9 @@ before_install:
 # By setting anything to org.apache.activemq.artemis.core.io.aio.AIOSequentialFileFactory.DISABLED we are disabling libaio loading on the testsuite
 script: 
 - set -e
-- mvn -Dorg.apache.activemq.artemis.core.io.aio.AIOSequentialFileFactory.DISABLED=AnythingNotNull -Pfast-tests -Pextra-tests -Ptests-CI -B install -q
+- ./mvnw -Dorg.apache.activemq.artemis.core.io.aio.AIOSequentialFileFactory.DISABLED=AnythingNotNull -Dnetty-transport-native-epoll-classifier=linux-`uname -m` -Pfast-tests -Pextra-tests -Ptests-CI -B install -q
```

Comment:
Can we keep this as plain mvn no need for mvnw

Reply:
There is no `mvn` pre-installed on the ARM64 VMs at TravisCI.

Reply:
There is no `mvn` pre-installed on ARM64 VMs at TravisCI